CHANGELOG.md

```diff
@@ -1,103 +1,127 @@
-## Master
+## [1.0.0.beta.5 2016.05.19](https://github.com/CocoaPods/CocoaPods-app/releases/tag/1.0.0.beta.8)
 
-* Update cask sparkle checkpoint on release  
+* Re-designed Home Window
+  [Orta Therox](https://github.com/orta)
+  [#285](https://github.com/CocoaPods/CocoaPods-app/pull/285)
+
+* Support using the keychain for access tokens for private git repos
+  [Orta Therox](https://github.com/orta)
+  [#279](https://github.com/CocoaPods/CocoaPods-app/pull/279)
+
+* SSL fixes
+  [David Ohayon](https://github.com/ohwutup)
+  [#281](https://github.com/CocoaPods/CocoaPods-app/pull/281)
+
+* Support for Handling Sources
+  [Nate West](https://github.com/nwest)
+  [#275](https://github.com/CocoaPods/CocoaPods-app/pull/275)
+
+* Update cask sparkle checkpoint on release
   [Samuel Giddins](https://github.com/segiddins)
   [#266](https://github.com/CocoaPods/CocoaPods-app/pull/266)
 
-* Show a warning while editing a Podfile for a Podfile.lock generated with a newer version of CocoaPods  
+* Show a warning while editing a Podfile for a Podfile.lock generated with a newer version of CocoaPods
   [Nate West](https://github.com/nwest)
   [#257](https://github.com/CocoaPods/CocoaPods-app/pull/257)
 
-* Migrate project to Xcode 7.3  
```

Comment:
argh, alright -  here you go @segiddins 

:P

![screen shot 2016-04-19 at 14 01 28](https://cloud.githubusercontent.com/assets/49038/14639970/3c757ff4-0637-11e6-9655-b0708671ac35.png)